recipe/meta.yaml

```diff
@@ -28,24 +27,38 @@ requirements:
     - python
     - bokeh >=3.1
     - colorcet
-    - matplotlib-base >=3.0
     - numpy >=1.21
     - packaging
     - pandas >=1.3
     - panel >=1.0
     - param >=2.0,<3.0
     - pyviz_comms >=2.1
+    - narwhals >=2
+    - python-dateutil >=2.8.2
   run_constrained:
     - plotly >=4.0
+    - matplotlib-base >=3
```

Comment:
We should keep this one in run. It has been included until now and people might get surprised if we remove it.